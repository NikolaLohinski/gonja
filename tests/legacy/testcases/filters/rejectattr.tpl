{{ [{'v': true}, {'v': false}, {'v': 0}, {'v': 1}]|rejectattr('v') }}
{{ [{'v': 0}, {'v': 1}, {'v': 2}, {'v': 3}, {'v': 4}, {'v': 5}]|rejectattr('v', 'odd') }}
{{ [{'v': 0}, {'v': 1}, {'v': 2}, {'v': 3}, {'v': 4}, {'v': 5}]|rejectattr('v', 'ge', 3) }}
