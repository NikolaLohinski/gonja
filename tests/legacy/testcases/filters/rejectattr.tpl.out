[{'v': False}, {'v': 0}]
[{'v': 0}, {'v': 2}, {'v': 4}]
[{'v': 0}, {'v': 1}, {'v': 2}]
